assessment.md

```diff
@@ -193,7 +193,36 @@
   * :sunglasses: = Polite, patient, and effectively communicates with technical
   leaders to seek advice and help when necessary
 
-## Revision of Guides
+### Revision of Guides
```

Comment:
I believe that the header should be ## instead of ###

Reply:
In the Markdown document Subcategories are denoted with three hashtags 
its the Overall category I.e. Technical or Professional that have two hashtags

Reply:
What you are saying is correct, however, `Revision of Guides` here is not a `subcategory` as I have found in this [version]<https://github.com/Allegheny-Computer-Science-203-S2020/cs203-S2020-assessment/commit/d73b1f9f70c527e9ca647000e017a3449543b05a> of the master branch. Thus, it should be in the same level as `Technical` or `Professional`

Reply:
Ohhh okay that makes sense, thanks Enpu!

Reply:
No problem!